vnext/Microsoft.ReactNative/ReactPackageBuilder.h

```diff
@@ -11,15 +12,18 @@ namespace winrt::Microsoft::ReactNative {
 struct ReactPackageBuilder : winrt::implements<ReactPackageBuilder, IReactPackageBuilder> {
   ReactPackageBuilder(
       std::shared_ptr<NativeModulesProvider> const &modulesProvider,
-      std::shared_ptr<ViewManagersProvider> const &viewManagersProvider) noexcept;
+      std::shared_ptr<ViewManagersProvider> const &viewManagersProvider,
+      std::shared_ptr<TurboModulesProvider> const &turboModulesProvider) noexcept;
```

Comment:
I think this is a publicly-visible breaking change too?

Reply:
This should be an internal class, since the idl file doesn't specify any constructor or implementing class.